Remove references to immutable in delegate-mtls-support.md (#5663)
harness/developer-hub: docs/platform/delegates/secure-delegates/delegate-mtls-support.md

## Move delegates to mTLS

You can move existing delegates (with the image tag in `yy.mm.xxxxx` format) to mTLS, legacy delegates are not supported. For more information on delegate types, go to [Delegate image types](/docs/platform/delegates/delegate-concepts/delegate-image-types).

### Prerequisites for delegate migration

Before you migrate an existing delegate to an mTLS-enabled delegate, make sure that you meet the following prerequisites.

> - The delegate must not be legacy delegate (it has image tag in _`yy.mm.xxxxx`_ format).

Reviewer: ```suggestion
- The delegate must have an image tag in _`yy.mm.xxxxx`_ format). It cannot be a legacy delegate.
```
